.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ env:
 on:
   pull_request:
   push:
-    branches: ['master', 'zio-2']
+    branches: ['series/1.x', 'series/2.x']
```

Comment:
Weren't we going to change this to just `series/2.x`?

Reply:
@adamgfraser I just applied this @sideeffffect proposal: https://github.com/zio/zio-prelude/pull/1048#pullrequestreview-1242270136

Reply:
@guizmaii I think based on that conversation we said we weren't going to include `series/1.x`.

Reply:
@adamgfraser Done :)